src/backend/cuda/CMakeLists.txt

```diff
@@ -301,13 +294,19 @@ if(UNIX)
       -Wl,--start-group
       ${CUDA_culibos_LIBRARY} #also a static libary
       ${CUDA_cublas_static_LIBRARY}
-      ${CUDA_cublasLt_static_LIBRARY}
       ${CUDA_cufft_static_LIBRARY}
-      ${CUDA_lapack_static_LIBRARY}
       ${CUDA_cusparse_static_LIBRARY}
       ${cusolver_static_lib}
       -Wl,--end-group
   )
+
+  if(CUDA_VERSION VERSION_GREATER 9.5)
```

Comment:
nit: I think a better check to avoid any unexpected issues (with decimals and all against different cmake versions) with cmake is to do
```cmake
if(CUDA_VERSION_MAJOR VERSION_GREATER 9)
```

On another note, I will take care of this with some other change.